react/training-project/src/app/components/Button/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button(props) {
```

Comment:
Turn it into a class component. Remember we use smar and /dumb components. For smart ones, we use class components, for dumb ones we use functions.